actors/builtin/miner/miner_actor.go

```diff
@@ -1245,8 +1246,7 @@ func terminateSectors(rt Runtime, sectorNos *abi.BitField, terminationType power
 	var dealIDs []abi.DealID
 	var allSectors []*SectorOnChainInfo
 	var faultySectors []*SectorOnChainInfo
-	var penalty abi.TokenAmount
-
+	penalty := abi.NewTokenAmount(0)
```

Comment:
This is the one I observed being evaluated. It's uninitialized if the termination type is `SectorTerminationExpired`.